# Sobre os leptóns

In [ ]:
# Configuración e imports. O código auxiliar está no paquete notebooks/fnyp/
%matplotlib inline
%reload_ext autoreload
%autoreload 2

import numpy as np
import matplotlib.pyplot as plt
import scipy.constants as units

try:
    from fnyp import common as fn
except ModuleNotFoundError:      # en Google Colab cómpre descargar antes o repositorio
    !git clone -q https://github.com/jahernando/USC-FNyP.git
    import sys; sys.path.append('USC-FNyP/notebooks')
    from fnyp import common as fn

fn.version()

## Obxectivos

   * Os leptóns senten a forza electrofeble e clasifícanse en tres familias que se comportan de forma universal.
   
   * As correntes cargadas presentan violación de carga e paridade. 
   
   * A violación de paridade está asociada á quiralidade dos espinores de Dirac.

   * Os neutrinos só senten a forza feble e, ao non teren masa, a súa helicidade e a súa quiralidade son equivalentes.

   * Os neutrinos oscilan entre os distintos sabores ao propagárense.
   

## Características dos leptóns

Os leptóns preséntanse en tres xeracións agrupados en dobletes:

$$
\begin{pmatrix}\nu_e \\ e \end{pmatrix}, \; \begin{pmatrix}\nu_\mu \\ \mu \end{pmatrix}, \;
\begin{pmatrix}\nu_\tau \\ \tau \end{pmatrix}
$$

As súas propiedades:

|       | $e$ | $\mu$ | $\tau$ | $\nu$ |
| :--   | :-- | :--   | :--    | :--   |
| masa  | 0.511 MeV    | 105.7 MeV  |  1777 MeV|   $\le$ 1 eV     |  
| carga |   -1  | -1 | -1 | 0 |
| vida media ($\tau$)| estable | $2.197$ $\mu\mathrm{s}$ | $290.6$ fs | estable|


O electrón e o neutrino son estables.

*Cuestión*: Cal é o percorrido medio dun $\mu$ ou dun $\tau$ con momento $p = 10$ GeV no baleiro?

Aplicando a transformación de Lorentz do tempo propio do sistema en repouso ao sistema do observador, obtemos:

$$
d = \, \gamma \, \beta \, \tau \, c, \;\; \gamma = \frac{E}{m}, \; \beta = \frac{p}{E}
$$

In [7]:
mass = 0.106 # GeV
tau  = 2.2 * units.micro # s
p    = 10 # GeV
ene  = np.sqrt(mass**2 + p**2)
gamma = ene/mass
beta  = p/ene
distance = gamma * beta * tau * units.c
print(' distancia {:e} m'.format(distance))

 distance 6.222108e+04 m


### Desintegracións

A seguinte táboa mostra as desintegracións principais dos leptóns:

|    | Fracción de desintegración |
|:-- | --:                                          |
| $\mu  \to e + \bar{\nu_e} + \nu_\mu$  | $\sim 100$ %|
| $\tau \to \mu + \bar{\nu_\mu} + \nu_\tau$| $17.4$ % |
| $\tau \to e + \bar{\nu_e} + \nu_\tau$| $17.8$ % |
| $\tau \to \mathrm{hadróns}+ \nu_\tau$| $\sim 64$ % |

que corresponden aos seguintes diagramas de Feynman:

| |
| :-- |
<img src="./imgs/feynman_lepton_decays.png" width = 500 align = "center">
|Desintegracións do $\mu$ (arriba) e do $\tau$ (abaixo)|

#### Conservación do número leptónico

Experimentalmente obsérvase que, en todas as interaccións coñecidas, se conserva o número leptónico total, definido como o número de leptóns menos o de antileptóns.

Ademais, esta conservación cúmprese independentemente en cada xeración: electrónica, muónica e tauónica.

Por convenio, asígnaselle a cada leptón un número leptónico $L=+1$ e á súa antipartícula $L=-1$, e de forma similar un número leptónico de sabor $L_e, L_\mu, L_\tau$ segundo a seguinte táboa:


|                                                 | $L_e$   | $L_\mu$ | $L_\tau$  |  $L$ |
| :--                                             | --      | --      | --        | --   |
| $(e, \nu_e), \; (e^+, \bar{\nu}_e)$            | +1, -1  |  0      | 0         | +1, -1 |
| $(\mu, \nu_\mu), \; (\mu^+, \bar{\nu}_\mu)$     | 0       |  +1,-1  | 0         | +1, -1 |
| $(\tau, \nu_\tau), \; (\tau^+, \bar{\nu}_\tau)$ | 0       |  0      | +1,-1     | +1, -1 |    

**O número leptónico de sabor** (por xeración) **e o total consérvanse.**


*Cuestión*: Verifica que o número leptónico de sabor e o total se conservan nas desintegracións de $\mu, \tau$.


Os experimentos que buscan violacións do número leptónico só lograron establecer límites inferiores á vida media dos procesos que o violarían, como na procura da desintegración:

$$
\mu \to e + \gamma
$$

A conservación da carga eléctrica garante que o electrón —a partícula cargada máis lixeira— sexa estable.

Porén, como veremos máis adiante, as oscilacións de neutrinos violan o número leptónico de sabor (electrónico, muónico ou tauónico), aínda que non o número leptónico total, e esta violación ocorre durante a propagación dos neutrinos.

*Cuestión*: A desintegración $\mu \to \nu_\mu + e + e^+$ viola algunha regra de conservación? E a desintegración $\tau \to \mu + \mu + \mu^+$?

### Descubrimentos dos leptóns
    
  * En 1932 C. Anderson descubriu **o positrón** nunha cámara de néboa nun campo magnético. Observou que entre os produtos dos raios cósmicos aparecían trazas que, pola súa curvatura, eran de electróns pero con carga positiva.
    
  * En 1937 Anderson e Neddermeyer descubriron partículas penetrantes nos raios cósmicos. En 1947 observáronse en emulsións fotográficas expostas a raios cósmicos as desintegracións do pión e do **muón**, que se correspondía coa partícula penetrante.
    
  * En 1975 M. Perl, no colisionador $e^+e^-$ de SLAC, descubriu **o tau** na produción de pares $e+e^+ \to \tau+\tau^+$, onde estes se desintegraban leptonicamente $\tau \to\mu +\bar{\nu}_\mu + \nu_\tau$
    

* En 1930, **Wolfgang Pauli** propuxo —como unha “solución desesperada”— a existencia do **neutrino**, unha partícula neutra, fermiónica e de masa extremadamente pequena, que escaparía á detección experimental. A súa hipótese resolvía o enigma da aparente non conservación da enerxía e do momento nas desintegracións $\beta$.

* En 1956, Frederick Reines e Clyde Cowan detectaron por primeira vez o antineutrino electrónico $\bar{\nu}_e$. Utilizando o intenso fluxo de antineutrinos emitido por un reactor nuclear en Savannah River (Carolina do Sur), observaron a interacción beta inversa:  $\bar{\nu}_e + p \to e^+ +n $, empregando un detector con líquido centelleador e auga como branco.

* En 1962, Jack Steinberger, Leon Lederman e Melvin Schwartz descubriron o neutrino muónico $\nu_\mu$, utilizando un feixe de neutrinos producido no AGS de Brookhaven (Nova York). O seu detector, composto por capas alternadas de material pasivo e cámaras de faíscas, permitiu demostrar que os $\nu_\mu$ son distintos dos $\nu_e$, e que cada tipo de neutrino aparece emparellado co seu leptón cargado correspondente ($e$ ou $\mu$).

* En 2001, a colaboración DONUT (Direct Observation of the NU Tau) en Fermilab observou por primeira vez a interacción dun neutrino tauónico $\nu_\tau$, mediante emulsións fotográficas de alta resolución.

Outros fitos importantes relacionados cos neutrinos:

* 1956 — C. S. Wu e colaboradores realizan o experimento que demostra a violación de paridade ($P$) nas desintegracións beta do $^{60}\mathrm{Co}$, confirmando a predición de Lee e Yang.

* 1958 — M. Goldhaber, L. Grodzins e A. Sunyar determinan que o neutrino electrónico posúe helicidade negativa, demostrando que é levóxiro.

* 1970 — R. Davis e J. Bahcall identifican o chamado problema dos neutrinos solares, ao observaren un fluxo de neutrinos procedentes do Sol moi inferior ao predito teoricamente.

* 1998 — T. Kajita e a colaboración Super-Kamiokande descobren as oscilacións de neutrinos atmosféricos, demostrando que os neutrinos posúen masa e cambian de sabor.

* 2001–2002 — A. McDonald e a colaboración SNO (Sudbury Neutrino Observatory) resolven o enigma dos neutrinos solares, confirmando as oscilacións e a conservación do número total de neutrinos.



### Descubrimento do neutrino

Na década de 1920, o espectro continuo de enerxía nas desintegracións $\beta$ presentábase como un auténtico misterio.

$$
^A_Z X \to ^A_{Z+1}X' + e \;\; (?)
$$

| |
|:--:|
|<img src="./imgs/intro_bspectrum_1935.png" width=400 align="center">|
| Espectro $\beta$|

Se o núcleo pai se atopa en repouso e só se emiten dúas partículas —o núcleo fillo e o electrón $\beta$—, entón o electrón debería ser monoenerxético, de acordo coa conservación do momento. A enerxía total dispoñible para ambos os produtos é $Q$, e no sistema centro de masas ambos son emitidos co mesmo momento $p^*$.

A situación era tan desesperada que N. Bohr chegou a comentar: _"At the present stage of atomic theory, however, we may say that we have no argument... for upholding the energy principle in the case of 𝛽-ray disintegrations.”_ («No estadio actual da teoría atómica, porén, podemos dicir que non temos ningún argumento... para manter o principio da enerxía no caso das desintegracións de raios 𝛽.»)


En 1930 [Pauli](https://en.wikipedia.org/wiki/Wolfgang_Pauli) postulou a existencia do neutrino nunha histórica carta.

| |
| :--: |
|<img src="./imgs/intro_Pauli_letter.jpg" width=600 align="center">|
||

Como un remedio desesperado para explicar o espectro continuo de enerxía na desintegración $\beta$, suxeriu a existencia dunha partícula neutra, fermiónica, penetrante, que escapase sen ser detectada e que, polo tanto, levase parte da enerxía da desintegración. Máis tarde Fermi denominaríaa neutrino.

Posteriormente B. Pontecorvo [[>]](https://www.youtube.com/watch?v=yXrHnsBgQSw&t=9s) suxeriu utilizar o intenso fluxo de neutrinos xerado nos reactores nucleares para detectalos mediante a interacción $\beta$ inversa:

$$
p + \bar{\nu}_e \to n + e^+
$$


En 1956 F. Reines e C. L. Cowan confirmaron experimentalmente a existencia do neutrino utilizando esta reacción. Empregaron o fluxo de antineutrinos, $\phi_\nu \simeq 10^{17}$ $\nu\mathrm{/(m^2 \, s)}$, xerado no reactor nuclear de Savannah River, Carolina do Sur, EUA.

*Cuestión*: Cal é a enerxía que debe ter o $\bar{\nu}_e$ para que ocorra esta interacción?

*Cuestión*: Debuxa o diagrama de Feynman correspondente.

| | |
| :--: | :--: |
| <img src="./imgs/intro_cowan_reines_detector.png" width=300 align="center"> | <img src="./imgs/leptons_cowanreines_drawing.png" width=320 align="center">|
| Detector Cowan-Reines | Esquema de detección [AB2.6]|

O sistema de detección empregaba un conxunto de centelleadores sensibles aos fotóns de 511 keV, producidos pola aniquilación do positrón, en coincidencia retardada cos fotóns gamma xerados na captura do neutrón en núcleos de Cd.

Esta dobre coincidencia caracterizaba inequivocamente o proceso de interacción beta inversa.

Reines e Cowan enviáronlle a Pauli o seguinte telegrama para anunciarlle o descubrimento do neutrino:

| | 
| :--: | 
| <img src="./imgs/intro_cowan_reines_telegram.png" width=350 align="center">|
| Telegrama que anuncia o descubrimento do neutrino|


[Para esta sección véxase AB2.2]

### Universalidade leptónica

A universalidade leptónica establece que a intensidade do acoplamento nos vértices das correntes febles cargadas é idéntica para as tres xeracións de leptóns.
Noutras palabras, os leptóns electrónicos, muónicos e tauónicos interactúan coa mesma intensidade a través do bosón $W^\pm$.

Imos comprobar, a partir dos datos experimentais, que esta igualdade de acoplamentos se cumpre con gran precisión.

Para iso, supoñeremos que a constante de acoplamento feble en cada vértice, denotada como $g_W^\alpha$ con $\alpha = e, \mu, \tau$, podería ser distinta para cada sabor leptónico, e contrastaremos esta hipótese cos resultados experimentais.

A anchura de desintegración do $\mu$, que se calcula nun curso máis avanzado, é:
 
$$
\Gamma (\mu \to e + \bar{\nu}_e + \nu_\mu) \equiv  \frac{1}{\tau_\mu} = \frac{G^2_F m^5_\mu}{192 \pi^3} = \frac{g^4_W}{32 m^4_W} \frac{m^5_\mu}{192 \pi^3}
$$

*Cuestión*: Verifica que as dimensións da fórmula anterior son correctas.



Na desintegración $\tau \to e + \bar{\nu}_e + \nu_\tau$ interveñen as constantes $g^\tau_W$ na corrente tauónica e $g^e_W$ na electrónica.

A anchura de desintegración parcial é:

$$
\Gamma (\tau \to e + \bar{\nu}_e + \nu_\tau) = \frac{(g^\tau_W)^2 (g^e_W)^2}{32 m^4_W} \frac{m^5_\tau}{192 \pi^3}
$$

A anchura de desintegración total é a suma das parciais, e a fracción de desintegración, $Br$, relaciónase coa anchura total e a parcial mediante:

$$
Br(\tau \to e + \bar{\nu}_e + \nu_\tau) = \frac{\Gamma(\tau \to e + \bar{\nu}_e + \nu_\tau)}{\Gamma_\tau} = \tau_\tau \, \Gamma(\tau \to e + \bar{\nu}_e + \nu_\tau)
$$

onde $\tau_\tau$ é a vida media do $\tau$.

Logo:

$$
\Gamma(\tau \to e + \bar{\nu}_e + \nu_\tau) = \frac{1}{\tau_\tau} \, Br(\tau \to e + \bar{\nu}_e + \nu_\tau) 
$$


A partir da razón entre as dúas anchuras:

$$
\frac{\Gamma(\tau \to \nu_\tau + e + \bar{\nu}_e)}{\Gamma(\mu \to \nu_\mu + e + \bar{\nu}_e)} = \frac{\tau_\mu}{\tau_\tau} \mathcal{Br}(\tau \to \nu_\tau + e + \bar{\nu}_e) = \frac{(g^\tau_W)^2 m^5_\tau}{(g^\mu_W)^2 m^5_\mu}
$$

podemos despexar a razón entre as constantes de cada sabor e, substituíndo polos seus valores experimentais, obtemos:

$$
\left( \frac{g^\tau_W}{g^\mu_W} \right)^2= \frac{m^5_\mu \tau_\mu}{m^5_\tau \tau_\tau} \, Br(\tau \to e + \bar{\nu}_e + \nu_\tau) = 1.0023 \pm 0.033
$$

Logo concluímos que hai unha única constante de acoplamento para as dúas familias, $g^\mu_W = g^\tau_W = g_W$.


In [8]:
tau_tau  = 290.6 * units.femto # s
mass_tau = 1777  # MeV
tau_mu   = 2.197 * units.micro # s
mass_mu  = 105.7   # MeV
br_tau_e = 0.178
ratio    = (tau_mu * mass_mu**5)/(tau_tau * mass_tau**5) * br_tau_e
print('Razón de anchuras {:4.3f}'.format(ratio))

Gamma ratio 1.002


Lembremos que a constante de Fermi $G_F$

$$
G_F = 1.166 \times 10^{-5} \;\; \mathrm{GeV}^{-2}.
$$

está relacionada coa intensidade do acoplamento $g_W$ do Modelo Estándar mediante:

$$
\frac{G_F}{\sqrt{2}} = \frac{g^2_W}{8 m^2_W},
$$

onde $m_W = 80.385 \pm 0.015$ GeV é a masa do $W$.

*Cuestión*: A partir das anchuras de desintegración $\Gamma(\tau \to \mu + \nu_\mu + \bar{\nu}_\tau)$ e $\Gamma(\tau \to e + \nu_e + \bar{\nu}_\tau)$, calcula $(g^e_W/g^\mu_W)^2$. 

*Nota*: A desviación respecto da unidade compénsase cun factor de espazo de fases.

[Para esta sección véxase AB7.2]

## Violación de paridade

Lembremos que a operación de inversión por paridade, $P$, cambia: 

$$
P  \, : \; {\bf x} \to - {\bf x}
$$

e lembremos tamén como se clasifican distintas magnitudes baixo paridade:

|        | Rango | Paridade | Exemplo |
|:--     | --    | -- | :-- |
| escalar | 0     | +1 | temperatura |
|pseudoescalar| 0 | -1 | helicidade |
| vector | 1 | -1   | momento |
|vector axial | 1 | +1 | momento angular, campo magnético | 

Na representación espinorial, a paridade vén dada pola matriz $\gamma^0$ [[véxase ext-trans-Lorentz]](ext-trans-Lorentz.ipynb)

$$
\Psi(t, {\bf x}) \to \gamma^0 \, \Psi(t, -{\bf x})
$$

### O experimento de Wu

Na década dos 50, Lee e Yang sospeitaron que as interaccións febles poderían violar a paridade. En 1957 Chien-Shiung Wu descubriu que as correntes cargadas non conservan a paridade. 

Wu mediu a dirección de emisión dos electróns na desintegración $\beta$ dunha mostra de $^{60}\mathrm{Co}$ a moi baixas temperaturas e baixo un campo magnético ${\bf B}$.

O $^{60}\mathrm{Co}$ desintégrase $\beta$:

$$
^{60}\mathrm{Co} \to ^{60}\mathrm{Ni}^* + e + \bar{\nu}_e.
$$


Wu mediu os electróns que saen cara arriba cando colocamos un campo magnético cara arriba ou cara abaixo. 

| |
| :-- |
| <img src="./imgs/leptons_Co_scheme.jpeg" width = 200 align = "center"> |
|Esquema da desintegración $\beta$ do $^{60}\mathrm{Co}$ polarizado|


Dicimos que un fenómeno viola a paridade se o fenómeno reflectido «no espello» non sucede coa mesma frecuencia ca o orixinal. Considera o seguinte gráfico: a dereita é o reflexo da esquerda.

| | 
| :--: | 
| <img src="./imgs/leptons_Co_spins.png" width = 400 align = "center"> | <
|Esquema da desintegración $\beta$ do $^{60}\mathrm{Co}$ polarizado|

O núcleo de $^{60}\mathrm{Co}$ ten un momento magnético ${\bf \mu}$, dado polo seu espín, ${\bf S}$, que está aliñado na dirección dun campo magnético ${\bf B}$. Tanto ${\bf B}$ coma ${\bf \mu}$ e ${\bf S}$ son vectores axiais, isto é, non cambian baixo paridade, mentres que o momento, ${\bf p}$, do electrón si cambia por ser un vector.

Se se conserva a paridade, o electrón debe saír coa mesma frecuencia no sentido da imaxe esquerda e no da dereita, isto é, con $\theta$ respecto á vertical e con $\pi-\theta$. No caso extremo, cando o electrón sae na vertical, o electrón sae coa mesma probabilidade cara arriba ca cara abaixo.

En xeral, os espíns dos núcleos dunha mostra están orientados en direccións arbitrarias. Necesitamos primeiro aliñalos.

M. Wu utilizou un campo magnético ${\bf  B}$ para orientar a dirección dos espíns dos núcleos (a través do seu momento angular) nunha dirección e colocou a mostra a moi baixas temperaturas para evitar que os espíns vibrasen polos efectos térmicos.

Wu fixou a posición do detector e cambiou o sentido do campo magnético, pero conceptualmente o experimento é equivalente ao gráfico anterior.

| | 
| :--: | 
| <img src="./imgs/leptons_Co_results.png" width = 360 align = "center"> |
| Razón de detección de $e$ en función do tempo e da polarización de ${\bf B}$ [WU]|


Na figura móstranse os resultados orixinais de Wu. Obsérvase que existe unha asimetría arriba–abaixo en función do tempo. A asimetría está presente cando os espíns están aliñados (mentres a mostra está fría) e vaise perdendo conforme os espíns se desaliñan (ao quentarse a mostra).

Apréciase que os electróns se emiten preferentemente no sentido oposto ao campo magnético, isto é, ao do espín do Co. Os electróns saen cara atrás. 

Poderiamos dicir que o norte o define o espín do núcleo, pero os electróns da desintegración $\beta$ prefiren dirixirse cara ao sur.

O experimento de Wu confirmou a predición de Lee e Yang —que recibirían o Premio Nobel, inxustamente sen incluíla a ela—: as desintegracións $\beta$ e, en xeral, *as correntes cargadas violan a paridade*.


| | 
| :--: | 
| <img src="./imgs/leptons_Wu_mirror.png" width = 600 align = "center"> |
| Chien-Shiung Wu co seu experimento (e diante do espello) |

*Cuestión:* Observa as imaxes. Unha é o reflexo da outra, pero só unha é real. Cal é a orixinal e cal o seu reflexo no espello? Que detalle che permite identificar a imaxe real?

### As correntes baixo paridade

A probabilidade de emisión do electrón nun determinado ángulo vén dada pola sección eficaz diferencial. Dado que a paridade non afecta á parte cinemática da sección eficaz, debe ser o elemento de matriz $M_{fi}$ o que cambie baixo paridade.

Porén, na teoría de Fermi, o elemento de matriz $M_{fi}$ preserva a paridade, dado que é proporcional ao produto de dúas correntes, a hadrónica e a leptónica:

$$
M_{fi} \propto  j_{\mathrm{had}}  \cdot  j_{\mathrm{lep}}
$$

Cada corrente cambia baixo paridade:

$$
P \; : \; j^\mu = (\rho, \, {\bf j}) \to (\rho, \, - {\bf j}),
$$

E o seu produto é, polo tanto, invariante.

$$
P \; : \; (\rho, \, {\bf j})_{\mathrm{had}} \cdot (\rho, \, {\bf j})_{\mathrm{lep}} = (\rho, \, - {\bf j})_{\mathrm{had}} \cdot (\rho, \,, - {\bf j})_{\mathrm{lep}}
$$


*Cuestión*: Aplica a paridade $P$ sobre a corrente fermiónica $\bar{\Psi} \gamma^\mu \Psi$.

A representación espinorial da paridade é simplemente: $P = \gamma^0$,

A corrente de probabilidade $j^\mu = \bar{\Psi} \gamma^\mu \Psi$ baixo paridade transfórmase en ${\bar \Psi} \gamma^0 \gamma^\mu \gamma^0 \Psi$, isto é:

$$
(j^0, {\bf j}) \to (j^0, -{\bf j}^k)
$$

polas propiedades de anticonmutación das matrices $\gamma^\mu$.

Neste punto temos que introducir a matriz $\gamma^5$:

$$
\gamma^5 = i \gamma^0 \gamma^1 \gamma^2 \gamma^3
$$

que na representación de Pauli-Dirac é:

$$
\gamma^5 = \begin{pmatrix} 0 & I \\ I & 0 \end{pmatrix}
$$

A matriz $\gamma^5$ cumpre as seguintes propiedades

$$
(\gamma^5)^2 = I, \;\;\; (\gamma^5)^\dagger = \gamma^5, \;\;\; \gamma^5\gamma^\mu = - \gamma^\mu \gamma^5
$$

*Cuestión:* Verifica as propiedades anteriores de $\gamma^5$.

Todas as posibles formas bilineais en $\Psi$ do tipo $\bar{\Psi} \Gamma \Psi$, onde $\Gamma$ son as posibles combinacións independentes que podemos crear coas matrices $\gamma^\mu$, son:

|        | -------- Forma -------- | Compoñentes | Espín |
|:--     | :--:    | :--: | :--: |
| escalar | ${\bar \Psi} \Psi$   | 1 | 0 |
|pseudoescalar|  ${\bar \Psi} \gamma^5 \Psi$  | 1 | 0|
| vector | ${\bar \Psi} \gamma^\mu \Psi$  | 4   | 1 |
|pseudovector | ${\bar \Psi} \gamma^\mu \gamma^5 \Psi$ | 4 | 1 | 
|tensor | ${\bar \Psi} (\gamma^\mu\gamma^\nu - \gamma^\nu \gamma^\mu) \Psi$ | 6 | 2 | 

En particular, vemos que hai unha forma, $\bar{\Psi}\gamma^\mu \gamma^5 \Psi$, que se comporta como un vector axial.
A presenza da matriz $\gamma^5$ transforma a forma escalar en pseudoescalar e a forma vector (ou corrente vectorial) en pseudovector (ou corrente axial).

Para definir o noso elemento de matriz $M_{fi}$ podemos considerar agora todas as combinacións das distintas formas que nos permitan obter un invariante escalar Lorentz.


A partir do estudo das desintegracións dos muóns, $\mu \to e + \nu_\mu + \bar{\nu}_e$, determinouse que formas interveñen no acoplamento das correntes febles cargadas.

| |
| :-- |
<img src="./imgs/leptons_espectrum_mudecay.png" width = 300 align = "center">
|Espectro de enerxía do electrón nas desintegracións do $\mu$; a liña é a teoría V-A [MU]|



A forma fenomenolóxica que corresponde ás desintegracións de muóns (e ás correntes cargadas) é:

$$
{\bar \Psi} \left[ \gamma^\mu \frac{1}{2}(I - \gamma^5) \right] \Psi
$$

Isto é, interveñen dúas correntes:

 * A corrente $j^\mu_V = \frac{1}{2}{\bar \Psi} \gamma^\mu \Psi$ é **vectorial** porque cambia como un vector fronte á paridade. 


* A corrente $j^\mu_A = \frac{1}{2}{\bar \Psi} \gamma^\mu \gamma^5 \Psi$ é **axial** porque cambia como un vector axial fronte á paridade. 

Por iso dicimos que as correntes cargadas teñen unha **estrutura $V-A$**

Como cambia agora $M_{fi}$ baixo paridade?

$$
P \, : \, j^\mu_V - j^\mu_A = (\rho_V - \rho_A, \; {\bf j}_V - {\bf j}_A) \to (\rho_V + \rho_A, - {\bf j}_V - {\bf j}_A)
$$

E agora $M_{fi}$, ao ser o produto de dúas correntes $V-A$, si que cambia baixo paridade:

$$
(\rho_V - \rho_A, \; +{\bf j}_V - {\bf j}_A)_{\mathrm{had}} \cdot (\rho_V - \rho_A, \; +{\bf j}_V - {\bf j}_A)_{\mathrm{lep}}  \\
\neq 
(\rho_V + \rho_A, \; -{\bf j}_V - {\bf j}_A)_{\mathrm{had}} \cdot (\rho_V + \rho_A, \; -{\bf j}_V - {\bf j}_A)_{\mathrm{lep}}
$$

### A quiralidade

En vez de ver a interacción como correntes $V-A$, podemos preguntarnos: que é o termo $\frac{1}{2} (I-\gamma^5)$ que aparece en ${\bar \Psi} \gamma^\mu \frac{1}{2}(I - \gamma^5) \Psi$?


Podemos observar que é un proxector que actúa sobre o espinor de Dirac:

$$
\Psi_L = \frac{1}{2}(I - \gamma^5) \, \Psi
$$

Existen dous proxectores complementarios:

$$
P_L \equiv \frac{1}{2} (I-\gamma^5) \,, \;\;\; P_R \equiv \frac{1}{2} (I+\gamma^5).
$$

que cumpren as condicións de proxectores:

$$
P_R + P_L = I, \;\; P^2_R = P_R, \;\; P^2_L = P_L, \; P_R P_L = P_L P_R = 0 
$$

Estes proxectores chámanse de **quiralidade** e proxectan o espinor en **levóxiro e dextróxiro**.

*Cuestión:* Verifica que $P_L, P_R$ cumpren as condicións anteriores.

Chamámolos proxectores de quiralidade porque un proxector se transforma no outro ao aplicarlle a inversión de paridade.

A representación espinorial da paridade é a matriz $P \equiv \gamma^0$; a transformación de paridade de $P_L$ é:

$$
P_L \to P^{-1} \, P_L \, P =  \gamma^0 \frac{1}{2}(1 - \gamma^5) \gamma^0 = \frac{1}{2} (1+\gamma^5) (\gamma^0)^2 = P_R 
$$



Deste xeito, un **espinor de Dirac** descomponse en dúas proxeccións, de **quiralidade levóxira e dextróxira**:

$$
\Psi = \Psi_R + \Psi_L 
$$
$$
\Psi_R = P_R \, \Psi, \;\;\; \Psi_L = P_L \, \Psi,
$$

que se intercambian ao aplicar a paridade:

$$
P \; : \;  \Psi_L \leftrightarrow \Psi_R
$$

Dicimos que unha é o reflexo da outra.

A quiralidade é unha **propiedade** dos espinores. Matematicamente, a quiralidade corresponde aos autovalores da matriz $\gamma^5$.

$$
\frac{1}{2}(1 \pm \gamma^5) \, \Psi_{R/L} = \Psi_{R/L} \;\; \Rightarrow \;\; \gamma^5 \, \Psi_{R/L} = \pm \, \Psi_{R/L}
$$

$\Psi_{R/L}$ son autoestados con autovalores $+1, -1$, respectivamente, de $\gamma^5$.


En xeral, cando falamos só de espinores, habitualmente omitimos a fase de $\Psi_{L/R}$ e escribimos principalmente a parte puramente espinorial, $u_{L/R}$


```{admonition} Nota
:class: note

Xa vimos que o espinor de Dirac presenta un comportamento fronte ás rotacións que desafía a nosa intuición clásica. Ao xiralo $2\pi$, o espinor cambia de signo; só tras un xiro completo de $4\pi$ recupera o seu estado inicial.

É o mesmo tipo de paradoxo visual que atopamos na formiga de Escher que percorre a cinta de Möbius: tras percorrer unha volta, a formiga atópase boca abaixo e ten que percorrer a cinta dúas veces para regresar á súa posición inicial.

```{admonition} Nota
:class: note

| | |
| :--: | :--: |
| <img src="./imgs/dirac_Escher_Moebius.jpg" width = 300 align = "center"> | <img src="./imgs/dirac_Escher_Moebius_pinv.jpg" width = 300 align = "center"> |
| Cinta de Möbius (esquerda) | Cinta de Möbius (dereita) |

De forma análoga, unha cinta de Möbius pode construírse en dúas versións quirais: unha obtida ao unir os extremos dunha tira cun xiro cara á esquerda, e a outra cun xiro cara á dereita.
Cada unha é a imaxe especular da outra, pero non poden superpoñerse mediante rotacións no espazo tridimensional.

Así como as dúas cintas de Möbius están relacionadas por unha reflexión, as dúas compoñentes do espinor —levóxira e dextróxira— conéctanse a través da operación de paridade.

*Exercicio:* Verifica que:

$$
\bar{\Psi} \gamma^\mu \Psi = \bar{\Psi}_L \gamma^\mu \Psi_L + \bar{\Psi}_R \gamma^\mu \Psi_R
$$

Podes verificar previamente que:

$$
\bar{\Psi} \gamma^\mu \frac{1}{2}(1-\gamma^5) \Psi = \bar{\Psi}_L \gamma^\mu \Psi_L
$$
$$
\bar{\Psi} \gamma^\mu \frac{1}{2}(1+\gamma^5) \Psi = \bar{\Psi}_R \gamma^\mu \Psi_R
$$


#### A quiralidade nas interaccións febles

Podemos dicir de forma equivalente que **as correntes cargadas**

* **son de tipo V-A**, $\; {\bar \Psi} \, \frac{1}{2}\gamma^\mu (I - \gamma^5) \, \Psi$ 

* ou que nelas **só interveñen os espinores levóxiros dos fermións**, $\Psi_L = \frac{1}{2} (I - \gamma^5) \, \Psi$, **e os dextróxiros dos antifermións**.


*Unha teoría que viole a paridade favorece unha quiralidade respecto á outra.*

```{admonition} Nota
:class: note

Podemos usar un símil coas cartas dunha baralla para comprender o efecto da quiralidade e da inversión de paridade.

As cartas teñen dúas caras, a da figura e o reverso, de xeito similar aos espinores, que teñen quiralidade levóxira e dextróxira. A inversión de cara dálle a volta á carta. A inversión por paridade cámbianos a quiralidade do espinor.

Un xogo de cartas invariante respecto á inversión de caras sería, por exemplo, simplemente contalas. Podemos lanzar as cartas sobre a mesa e contalas; dá igual que estean boca arriba ou boca abaixo. Nada cambia se lle damos a volta a cada unha das cartas. Algo moi aburrido. Os xogos habituais de cartas si violan a inversión de caras porque só usamos a cara da figura. Aí está parte da diversión!

Algo así sucede na Natureza: os espinores teñen dúas quiralidades e a inversión de paridade inverte a quiralidade. Nas interaccións electromagnéticas e fortes non hai distinción entre as quiralidades dos espinores, preservan a paridade, pero si hai distinción nas interaccións febles. Nas correntes febles cargadas (mediadas polos $W^\pm$) só participan os espinores levóxiros das partículas e os dextróxiros das antipartículas, isto é, víolase a paridade de forma máxima.

```{admonition} Nota
:class: note

Os físicos do século XX utilizaron definicións que levan á confusión, como o abuso dos diversos "isospíns", pero hai que ter especial coidado en non confundir a *helicidade* coa *quiralidade*.

Outro punto de confusión vén dado porque, cando traballamos con espinores directamente (isto é, cando non traballamos directamente en teoría cuántica de campos), o resultado físico dalgúns operadores sobre as antipartículas require inverter o signo, como é de novo o caso da quiralidade. Para a parte espinorial das antipartículas, $v_R$

$$
v_{R/L} = \mp \gamma^5 v_{R/L} 
$$

Desafortunadamente, deste xeito, a proxección de quiralidade dextróxira do espinor $v$ vén dada por:

$$
v_R = \frac{1}{2}(1-\gamma^5) \, v
$$


### Helicidade e quiralidade

Aínda que *helicidade* e *quiralidade* non son o mesmo, hai un caso moi relevante no que coinciden.

Lembremos que a helicidade é a proxección do espín na dirección do momento da partícula.

Na representación de Pauli-Dirac, os proxectores de quiralidade son simplemente:

$$
P_R = \frac{1}{2}\begin{pmatrix} I & I \\ I & I \end{pmatrix}, \;\;\;
P_L = \frac{1}{2}\begin{pmatrix} I & -I \\ -I & I \end{pmatrix},
$$


Sexan agora os espinores de helicidade $\pm$ con ${\bf p} = p \, \hat{k}$ na dirección $z$:

$$
u_{+} = N \begin{pmatrix} 1 \\ 0 \\ \kappa  \\ 0 \end{pmatrix}, \;
u_{-} = N \begin{pmatrix} 0 \\ 1 \\ 0 \\ -\kappa  \end{pmatrix}
$$

onde $N$ é un factor de normalización e:
 
$$
\kappa = \frac{p}{E+m}
$$


Se aplicamos os proxectores de quiralidade sobre os espinores de helicidade $u_{\pm}$, obtemos:

$$
P_R \, u_+ = \frac{1}{2}(1+\kappa) N \begin{pmatrix} 1 \\ 0 \\ 1  \\ 0 \end{pmatrix}, \;\;\;
P_L \, u_+ = \frac{1}{2}(1-\kappa) N \begin{pmatrix} 1 \\ 0 \\ -1  \\ 0 \end{pmatrix}, \\
P_R \, u_- = \frac{1}{2}(1-\kappa) N \begin{pmatrix} 0 \\ 1 \\ 0  \\ 1 \end{pmatrix}, \;\;\;
P_L \, u_- = \frac{1}{2}(1+\kappa) N \begin{pmatrix} 0 \\ 1 \\ 0  \\ -1 \end{pmatrix}, \;\;\;
$$


Obtemos así a táboa cos coeficientes das compoñentes de quiralidade dos espinores $u$ de cada helicidade:

| helicidade $\downarrow$ \ quiralidade  $\rightarrow$| --- R --- | --- L --- |
| :--: | :--: | :--: | 
| + | $\frac{1}{2}(1+\kappa)$ | $\frac{1}{2}(1-\kappa)$ |
| - | $\frac{1}{2}(1-\kappa)$ | $\frac{1}{2}(1+\kappa)$ |
| nos acoplamentos co $W$  | antifermións, $v_R$     | fermións, $u_L$ |

Dixemos que nas interaccións febles cargadas só participan os espinores $u_L$, isto é, a última columna, L.

Se repetísemos os cálculos para os espinores $v_R$, obteriamos os mesmos coeficientes. Nas correntes febles entraría a parte $v_R$, que correspondería á columna do medio, R.


Consideremos agora o caso de partículas sen masa, $m=0$, ou ultrarrelativistas, $m \ll E$; en ambos os casos temos que $\kappa = 1$ e, polo tanto:

| helicidade $\downarrow$ \ quiralidade  $\rightarrow$ | --- R --- | --- L --- |
| :--: | :--: | :--: | 
| + | 1 | 0 |
| - | 0 | 1 |
| nos acoplamentos co $W$    | antifermións, $v_R$     | fermións, $u_L$ |

Isto é, **para partículas sen masa ou ultrarrelativistas, a quiralidade e a helicidade coinciden**.

Nestes dous casos -que son os máis comúns a altas enerxías- só participan nas correntes cargadas os espinores dos fermións $u_-$ con helicidade negativa e os dos antifermións $v_+$ con helicidade positiva.


## O neutrino nas correntes cargadas

### Helicidade do neutrino

En 1957, Maurice Goldhaber e os seus colaboradores mediron no laboratorio de Brookhaven (Nova York, EUA) a helicidade do neutrino e descubriron que é sempre negativa. Este constitúe un dos experimentos máis enxeñosos e sutís da historia da Física de Partículas.

Goldhaber estudou a emisión de fotóns na seguinte captura electrónica:

$$
\mathrm{Eu}(J=0) + e \to \mathrm{Sm}^*(J=1) + \nu_e \to \mathrm{Sm}(J=0) + \gamma
$$

Para iso empregaron un material inusual: europio radiactivo, cuxos electróns de captura foron polarizados mediante un campo magnético externo. O deseño experimental garantía que neutrinos e fotóns se emitisen preferentemente ao longo da dirección vertical.

Inicialmente, o núcleo de europio posúe momento angular total nulo. Tras a captura electrónica, o núcleo de samario queda excitado con momento angular $J=1$ e desintégrase case de inmediato ao seu estado fundamental ($J=0$), emitindo un fotón gamma de espín 1.

| |
| :-- |
<img src="./imgs/leptons_goldhaber_experiment.png" width = 200 align = "center">
|Esquema do experimento de Goldhaber [GOL]|


Goldhaber mediu o número de fotóns emitidos —proporcional ao número de capturas electrónicas— nunha dirección e sentido determinados (neste caso, cara abaixo), para dúas orientacións opostas do campo magnético: cara arriba e cara abaixo.

No esquema da seguinte figura móstrase o electrón inicial co seu espín polarizado en ambas as direccións, así como a emisión do neutrino e do fotón ao longo do eixe vertical. Tamén se representan os seus espíns e, polo tanto, a helicidade correspondente de cada partícula.

| |
| :-- |
<img src="./imgs/leptons_goldhaber_spins.png" width = 200 align = "center">
|Helicidades no experimento de Goldhaber|


A dirección $z$ defínese como a “vertical” do experimento, determinada polo eixe do campo magnético ${\bf B}$, que polariza o electrón procedente da captura de $^{152}\mathrm{Eu}$ cara arriba ou cara abaixo, con $S_z(e) = \pm \tfrac{1}{2}$ segundo a orientación do campo.

Selecciónanse os fotóns que se emiten cara abaixo, e a configuración experimental garante que o neutrino correspondente se emite cara arriba. A terceira compoñente do espín, $S_z = \pm \tfrac{1}{2}$, consérvase no proceso, o que implica que a helicidade do neutrino coincide coa do fotón emitido.

Se se observase o mesmo número de fotóns en ambas as configuracións, o neutrino posuiría as dúas helicidades posibles. En cambio, se unha das configuracións domina, significaría que o neutrino ten unha helicidade preferente.

O resultado do experimento de Goldhaber demostrou que os neutrinos posúen só helicidade negativa, é dicir, a proxección do seu espín sobre a dirección do momento é sempre oposta a este.


A seguinte táboa mostra as posibilidades de espín e as helicidades posibles no experimento de Goldhaber.

| ----- $S_z(e)$ -----  | ----- $S_z(\nu)$ ----- | ----- $S_z(\gamma)$ ----- | ----- $h(\nu)$ ----- | ----- $h(\gamma)$ -----|  observación |
| :--:   | :--:        | :--:         |:--:           | :--:         | :--: |
|  1/2   |  -1/2  |  1  |  -   |  -      |  si|
|-1/2   |  1/2  |  -1  | +    |+     |  non|

Na Natureza só se observa a primeira posibilidade: existen unicamente neutrinos con helicidade negativa; os de helicidade positiva non se producen.

Xa vimos que, para unha partícula sen masa, helicidade e quiralidade coinciden, de modo que o feito de que os neutrinos teñan helicidade negativa implica que posúen quiralidade levóxira.

Este resultado está intimamente ligado ao principio fundamental de que as interaccións febles cargadas só actúan sobre fermións con quiralidade levóxira (e sobre antifermións con quiralidade dextróxira).

### A masa do neutrino

Os experimentos que intentaron medir directamente a masa do neutrino só conseguiron establecer límites superiores ao seu valor.

A masa do neutrino búscase a partir da distorsión no extremo final do espectro de enerxía do electrón na desintegración $\beta$ do tritio:

$$
^3\mathrm{H} \to ^3\mathrm{He} + e + \bar{\nu}_e,
$$

| |
| :-- |
<img src="./imgs/leptons_numass_tritium.jpeg" width = 500 align = "center">
|Espectro de enerxía do electrón da desintegración do tritio se o neutrino ten unha masa $\sim$ 1 eV|

Se o neutrino posúe masa, o electrón non pode levar toda a enerxía dispoñible na desintegración, polo que o extremo final do espectro de enerxía do electrón se ve modificado de forma dependente da masa do neutrino.

O experimento KATRIN, actualmente en funcionamento, estableceu o límite máis baixo ata a data para a masa do neutrino:

$$
m_{\nu_e} < 0.45 \; \mathrm{eV}
$$

O neutrino é -polo menos- un millón de veces máis lixeiro ca o electrón!



### Espinores de Weyl

Imos ver que o espinor de Dirac dos neutrinos —se estes non teñen masa— pode descompoñerse en dous biespinores independentes, chamados espinores de Weyl, cada un cunha quiralidade distinta.

Utilizaremos para iso a representación de Weyl (ou quiral) das matrices de Dirac:

$$
\gamma^0 = \begin{pmatrix} 0 & I \\ I & 0 \end{pmatrix}, \;\;
\gamma^k = \begin{pmatrix} 0 & \sigma^k \\ -\sigma^k & 0 \end{pmatrix}, \;\;
\gamma^5 = \begin{pmatrix} -I & 0 \\ 0 & I \end{pmatrix}
$$

Deste xeito, os operadores de proxección de quiralidade son:

$$
P_L = \begin{pmatrix} I & 0 \\ 0 & 0 \end{pmatrix}, \;
P_R = \begin{pmatrix} 0 & 0 \\ 0 & I \end{pmatrix} \;
$$

*Cuestión:* Verifica que as matrices da representación de Weyl cumpren as condicións das matrices $\gamma^\mu$.

Nesta representación, o espinor de Dirac está formado por dous **biespinores**, $u_L, u_R$, chamados de Weyl, que corresponden á parte de quiralidade levóxira e dextróxira do espinor de Dirac.

$$
u(p) = \begin{pmatrix} u_L(p) \\ u_R(p)\end{pmatrix}
$$

Podemos comprobar que as proxeccións de quiralidade son:

$$
P_L \, u = \begin{pmatrix} I & 0 \\ 0 & 0 \end{pmatrix} \, \begin{pmatrix} u_L \\ u_R\end{pmatrix} = \begin{pmatrix} u_L \\ 0 \end{pmatrix}, \;\;\;\,
$$

$$
P_R \, u =  \begin{pmatrix} 0 & 0 \\ 0 & I \end{pmatrix} \, \begin{pmatrix} u_L \\ u_R\end{pmatrix} = \begin{pmatrix} 0 \\ u_R\end{pmatrix}.
$$

E que a paridade intercambia as quiralidades

$$
P \to \gamma^0 \, u =  \begin{pmatrix} 0 & I \\ I & 0 \end{pmatrix} \, \begin{pmatrix} u_L \\ u_R\end{pmatrix} = \begin{pmatrix} u_R \\ u_L\end{pmatrix} 
$$


Se agora consideramos o caso revelador dun *fermión sen masa, $m=0$*, a ecuación de Dirac redúcese a:

$$
\gamma^\mu p_\mu \, u(p) = 0
$$

E descomponse en dúas ecuacións independentes para os biespinores de quiralidade dextróxira e levóxira:

$$
(E - {\bf p} \cdot \vec{\sigma}) \, u_R(p) = 0, \\
(E + {\bf p} \cdot \vec{\sigma}) \, u_L(p) = 0.
$$

É dicir, o espinor de Dirac sepárase en dous espinores de Weyl independentes, cada un cunha quiralidade definida.

Dado que para unha partícula sen masa se cumpre $E = p$, onde $\mathbf{p} = p, \hat{\mathbf{v}}$, as ecuacións anteriores poden escribirse como:

$$
(\hat{\bf v} \cdot \vec{\sigma}) \; u_R(p) = + u_R(p), \\ (\hat{\bf v} \cdot \vec{\sigma}) \; u_L(p) = - u_L(p).
$$

É dicir, existe unha correspondencia biunívoca entre helicidade e quiralidade para os fermións sen masa: o espinor de Weyl de quiralidade levóxira posúe helicidade negativa, e o de quiralidade dextróxira, helicidade positiva:

$$
u_R = u_+, \;\;\; u_L = u_-
$$

Neste caso, quiralidade e helicidade son propiedades equivalentes.

Ademais, os biespinores de Weyl son completamente independentes, xa que a ecuación de Dirac se separa en dúas ecuacións desacopladas, unha para cada quiralidade.

### O neutrino no Modelo Estándar

*O Modelo Estándar postula que o neutrino non ten masa*, o que implica que os seus espinores de quiralidade levóxira e dextróxira son independentes.

Os neutrinos só interaccionan debilmente, e *nas interaccións febles só interveñen as proxeccións levóxiras* (dextróxiras para as antipartículas), isto é, $u_L$ e $v_R$. *Só son reais eses dous espinores* para o neutrino:

$$
u_L \equiv u_-, \;\;\; v_R \equiv v_+
$$

En termos da conxugación de carga e da paridade, atopamos:

| |      |  $\hat{P}$| |
| :--: | :--: | :--: | :-- |
|  | $\nu_L$| $\Rightarrow$ | $\nu_R$  (non existe)|
| $\hat{C}$ | $\Downarrow$|  |  $\Downarrow$|
|  | $\bar{\nu}_L$ (non existe) | $\Rightarrow$| $\bar{\nu}_R$ 

**A forza feble viola de forma máxima a paridade, $P$, e a conxugación de carga, $C$, pero non $CP$**.


Dado que **o neutrino**:

  * **ten masa compatible con cero**,
  
  * **ten helicidade negativa** (o antineutrino, positiva),
  
**o Modelo Estándar postula** que **os neutrinos non teñen masa** e só existen con **quiralidade levóxira**, e o antineutrino con quiralidade dextróxira, 

$$
\nu_L, \;\;\; {\bar \nu}_R.
$$ 


**As correntes cargadas nos leptóns violan a paridade, P, e a carga, C, pero preservan CP**

| |
| -- |
 <img src="./imgs/leptons_escher.jpg" width = 450 align = "center"> |

Efecto de aplicar a paridade (reflexo, esquerda-dereita) e a carga (inverter os grises, arriba-abaixo) nun debuxo inicial de M.C. Escher. 


```{admonition} Nota
:class: note

Os físicos de partículas sentíronse reconfortados -aínda que só por un breve período- ao comprobaren que, se ben a inversión por paridade, $P$, e a inversión por carga, $C$, se violaban, a combinación conxunta $CP$ conservábase.

Naquela época, a mediados do século XX, cando os extraterrestres estaban de moda, Feynman facíase a seguinte pregunta: como podería estar seguro de estreitarlle a man a un alieníxena que quizais estivese composto só de antipartículas? 

E el mesmo respondía: pediríalle que repetise o experimento da señora Wu, que me dixese a dirección privilexiada cara á que saen os electróns do seu mundo nunha desintegración $\beta$ respecto ao espín do núcleo. Se no seu mundo, coma no noso, saen na dirección oposta, dámoslle a man; se saen na mesma... mellor non.

## Desintegración do pión

| |
| -- |
<img src="./imgs/feynman_pion_decay.png" width = 300 align = "center">
|Diagrama de Feynman de $\pi^+ \to l^+ + \nu_l$|

A desintegración do pión $\pi^+ \to l^+  + \nu_l$ exemplifica os efectos da estrutura V-A das correntes cargadas.

A razón experimental das anchuras de desintegración a $l = \mu, e$ é:

$$
\frac{\Gamma(\pi^+ \to e^+ + \nu_e)}{\Gamma(\pi^+ \to \mu^+ + \nu_\mu)} = 1.230 \pm 0.004 \; 10^{-4}
$$

Porén, o espazo de fases dispoñible para o $e$ é maior ca o do $\mu$, dado que a razón entre eles vén dada por $p^*_e/p^*_\mu$, e o momento $p^*_e > p^*_\mu$.

Por que, entón, o $\pi^+$ se desintegra principalmente a $\mu^+$ e non a $e^+$?

*Cuestión:* Calcula $p^*_l, E^*_l$ no CM.

*Cuestión:* Calcula a razón $p^*_e/p^*_\mu$.

No CM
$$
E_l = \frac{m^2_\pi + m^2_l}{2 m_\pi}, \; \; p_l = \frac{m^2_\pi - m^2_l}{2 m_\pi}
$$

In [2]:
mpi = 139.51  # MeV
mmu = 105.66  # MeV
me  =   0.511 # MeV
ene_cm = lambda ml : (mpi**2 + ml**2)/(2*mpi)
p_cm   = lambda ml : (mpi**2 - ml**2)/(2*mpi)
a_R    = lambda ml : (1 - p_cm(ml)/(ene_cm(ml) + ml))/2

beta_mu, beta_e = p_cm(mmu)/ene_cm(mmu), p_cm(me)/ene_cm(me)
espace_phase    = p_cm(me)/p_cm(mmu)
print(' beta mu    = {:7.6f}   , beta    e = {:7.6f}    '.format(beta_mu, beta_e))
print(' enerxía mu = {:5.3f} MeV, enerxía e = {:5.3f} MeV'.format(ene_cm(mmu), ene_cm(me)))
print(' espazo de fases          = {:e}'.format(espace_phase))

 beta mu   = 0.270970   , beta   e = 0.999973    
 energy mu = 109.767 MeV, energy e = 69.756 MeV
 espacio fásico          = 2.345194e+00


Vemos que o $\mu$ non é relativista, $\beta = 0.27$, pero o electrón si, $\beta = 0.99997$. 


Revisemos agora a helicidade das partículas da desintegración.

| |
| :-- |
<img src="./imgs/leptons_piondecay_spins.png" width = 300 align = "center">
|Momento e espín de $\pi^+ \to l^+ + \nu_l$ no CM|

No CM, o $\pi^+$ ten espín nulo, o neutrino, $\nu_l$, ten necesariamente helicidade (-) e é levóxiro $(L)$, e para conservar o momento angular, o antileptón cargado, $l^+$, debe ter helicidade (-). A proxección do espín do antileptón cargado é oposta á do neutrino para que a suma total do espín sexa nula.


Por outra banda, sabemos que só a proxección con quiralidade dextróxira dos antileptóns intervén na desintegración. Canto vale o coeficiente asociado a esa proxección para o $e^+$ e o $\mu^+$?

Lembremos que o espinor $v_-$ con helicidade (-) ten como coeficiente de quiralidade dextróxira:

$$
\frac{1}{2}(1-\kappa) = \frac{1}{2}\left(1 - \frac{p_l}{E_l + m_l} \right)
$$

A partir de $E_l, p$ no CM, obtemos:

$$
\frac{1}{2}(1-\kappa) = \frac{m_l}{m_\pi + m_l}
$$

O factor $(1-\kappa)/2$ é moito máis pequeno para o $e$ ca para o $\mu$.


O factor asociado ás proxeccións dextróxiras entre o $e$ e o $\mu$ que aparece en $|M_{fi}|^2$ será:

$$
\frac{m^2_e}{m^2_\mu} \frac{(m_\pi + m_e)^2}{(m_\pi + m_\mu)^2}\sim 7 \, 10^{-5}
$$

A desintegración do $\pi^+$ a $e^+$ está moi suprimida porque a compoñente de quiralidade dextróxira do $e^+$ está moi suprimida para un $e^+$ de helicidade negativa. Neste caso, o feito de que o electrón sexa relativista e a conexión helicidade-quiralidade son a razón pola que esta desintegración está moi suprimida.

Deste xeito, a preponderancia da desintegración do $\pi^+$ a $\mu^+$ explícase pola natureza $V-A$ das interaccións febles cargadas.

In [3]:
helicity_supres = a_R(me)/a_R(mmu)
print(' supresión por helicidade = {:e}'.format(helicity_supres**2))
print(' total                    = {:e}'.format(espace_phase * helicity_supres**2))

 supresion por helicidad = 7.170821e-05
 total                   = 1.681697e-04


Na seguinte figura móstrase a distribución de enerxía do espectro de electróns nas desintegracións do $\pi^+$.

| |
| :-- |
<img src="./imgs/leptons_espectrum_pidecay.png" width = 400 align = "center">
|Espectro de enerxías do $e$ nas desintegracións do $\pi$ [AB]|

Pode observarse que a desintegración do $\pi$ a $e$ está suprimida (hai menos sucesos na distribución), e tamén que o electrón é monoenerxético (a súa enerxía é un pico); mentres que os electróns que proveñen da desintegración do $\mu$ son máis abundantes e presentan unha distribución continua en enerxía.

[Para esta sección véxase MT11.2.3, MT11.6, AB7.3, AB7.4, AB7.5]

## Oscilacións de neutrinos

Sabemos experimentalmente que **os neutrinos oscilan entre os seus sabores na súa propagación**.

As oscilacións **dependen das masas** dos neutrinos.

Pero lembremos que o Modelo Estándar postula que os neutrinos non teñen masa.

Isto fai que as oscilacións de neutrinos sexan polo de agora a **única proba experimental** de que **o SM necesita revisarse**.



### O quebracabezas solar

Na década dos 60, o experimento de R. Davis na mina de Homestake, Dakota do Sur, mediu o fluxo de $\nu_e$  solares e resultaba ser $1/3$ do esperado segundo o modelo teórico do Sol desenvolvido por J. Bahcall.

Durante aproximadamente 40 anos os resultados dos experimentos que medían o fluxo de $\nu_e$ solares mostraban un déficit respecto do fluxo esperado segundo o modelo solar.

En 2004 o experimento SNO (en Canadá) mediu o fluxo total (a suma dos fluxos dos diferentes sabores) de neutrinos e verificou que estaba de acordo co esperado polo modelo solar para os neutrinos, todos $\nu_e$, producidos no Sol. Polo tanto, os neutrinos que chegan á Terra corresponden cos emitidos polo Sol, só que alí prodúcense como $\nu_e$ e na Terra detéctase só 1/3 con ese sabor; os 2/3 restantes cambian de sabor.

Pontecorvo, nos anos 50, anticipou que os neutrinos poderían oscilar entre os seus diversos sabores se tiñan masas distintas.



### Neutrinos atmosféricos

En 1998 o experimento Super-Kamiokande, na mina de Kamioka, Xapón, mediu que o fluxo de $\nu_\mu$ producido nas interaccións dos raios cósmicos coa atmosfera dependía da dirección de chegada; en concreto, mediu que existía unha asimetría arriba-abaixo, onde abaixo correspondía aos neutrinos que atravesaran a Terra, e arriba só a atmosfera.

Agora sabemos que os $\nu_\mu$, dependendo da distancia percorrida, transfórmanse de sabor nos outros dous neutrinos, pero principalmente en $\nu_\tau$.

A seguinte figura mostra o detector Super-Kamiokande.

| |
| :-- |
<img src="./imgs/leptons_SK.png" width = 460 align = "center">
|Super-Kamiokande a) debuxo do detector b) principio de detección [SK, MT13.2]|

SK é un tanque de auga de 50 kton, 36 m de altura e 34 de diámetro, con 11 k PMT nas paredes.


SK detecta as correntes cargadas de neutrinos $\nu_e \to e + X$ ou $\nu_\mu \to \mu + X$ a partir do leptón cargado.

Na auga, o leptón, a partir dun limiar de enerxía, emite luz Cherenkov en forma de cono, que se detecta en xigantescos PMT colocados nas paredes do detector.

A partir da luz e do tempo da súa chegada aos PMT determínanse a enerxía e a dirección do neutrino.

Pódense distinguir $e$ de $\mu$ polos patróns de luz nos PMT.

Resultados da colaboración SK.

| |
| :-- |
<img src="./imgs/leptons_SK_results.png" width = 420 align = "center">
|sucesos de $\nu_e$ (esq.) e $\nu_\mu$ (dta.) detectados por SK en función do ángulo [SK]|


Na figura, o eixe $x$ corresponde ao $\cos \theta$, onde $\theta$ é o ángulo de chegada; $\cos \theta = 1$ indica arriba, e $\cos \theta = -1$, abaixo (é dicir, atravesando a Terra).

Observamos:

 * Para os $\nu_e$ a predición e a observación coinciden.

 * Para os $\nu_\mu$ de multi-GeV hai un déficit para os que proveñen de 'abaixo'.

 * A liña corresponde á predición no caso de oscilacións.


 Logo SK concluíu que os neutrinos $\nu_\mu$ producidos na atmosfera oscilan no seu percorrido ata o detector.

### Oscilacións en dúas familias

Como os neutrinos son neutros, un neutrino dun sabor, por exemplo $\nu_e$, pode estar composto por unha combinación de neutrinos con diferentes masas.

Cada neutrino propagarase libremente de acordo coa súa masa como unha onda libre; e coa distancia -ou o tempo transcorrido-, a composición inicial dos diferentes tipos de neutrinos cambia.

Os neutrinos poden oscilar se os estados de sabor non son os mesmos cós estados de masa e as masas dos neutrinos son distintas.

Os estados de sabor son $\nu_\alpha$, con $\alpha = e, \mu, \tau$, e os estados de masa $\nu_i$ con masas $m_i$  onde $i = 1, 2, 3$.

Consideremos por simplicidade só dous sabores e dúas masas. Os sabores serán: $\alpha, \beta$

A relación entre estados de masa e de sabor dos neutrinos vén dada por unha matriz de mestura:

$$
\begin{pmatrix} \nu_\alpha \\ \nu_\beta \end{pmatrix} = \begin{pmatrix} \cos \theta & \sin \theta \\ - \sin \theta & \cos \theta \end{pmatrix} \begin{pmatrix} \nu_1 \\ \nu_2 \end{pmatrix}
$$


Se producimos un neutrino dun determinado sabor, $\nu_\alpha$, nun tempo $t_0  = 0$, cun momento $p$, este é unha combinación dos neutrinos de masa $| \nu_1 \rangle, \, | \nu_2 \rangle$ que depende dun ángulo $\theta$.

$$
| \nu_\alpha \rangle = \cos \theta | \nu_1 \rangle + \sin \theta | \nu_2 \rangle
$$

A evolución de cada neutrino de masa corresponde á dunha partícula libre, isto é:

$$
| \nu (t) \rangle  = \cos \theta e^{+i({\bf p x} - E_1t)}| \nu_1 \rangle + \sin \theta e^{+i({\bf p x} - E_2t)}| \nu_2 \rangle
$$

onde $E_i = \sqrt{m^2_i + p^2}$, con $i = 1, 2$


Nun determinado tempo $t$, ou equivalentemente a unha distancia $L$, podemos detectar o neutrino con outro sabor $\beta$.

$$
| \nu_\beta \rangle = -\sin \theta | \nu_1 \rangle + \cos \theta | \nu_2 \rangle
$$

Lembremos que as interaccións dos neutrinos teñen lugar nos seus estados de sabor!

Calculamos a amplitude de transición:

$$
A_{\alpha \beta}(t) = \langle \nu_\beta | \nu(t) \rangle = -\sin\theta\cos\theta e^{-iE_1t} + \sin\theta\cos\theta e^{-iE_2t}
$$

lembremos que unha fase global na amplitude, neste caso $e^{i{\bf px}}$, non afecta á física do proceso.

Se operamos:
$$
A_{\alpha\beta}(t) = \frac{1}{2}\sin 2 \theta e^{-iE_1} \left( 1 + e^{-i(E_2 - E_1)t}\right)
$$

de novo a fase global $e^{-iE_1t}$ non altera a física, e se desenvolvemos:

$$
E_2 - E_1 \simeq \frac{p}{2}  \left(1 + \frac{m_2^2}{p^2}\right) - \frac{p}{2}  \left(1 + \frac{m_1^2}{p^2}\right) = \frac{m_2^2 - m^2_1}{2p} \simeq \frac{\Delta m^2_{21}}{2E}
$$

onde $\Delta m_{21}^2 = m_2^2 - m^2_1$, e dado que $m_i \sim 0$ aproximamos $p \sim E$


A probabilidade de aparición, ou de oscilación, dun $\nu_\alpha$ como $\nu_\beta$ a un tempo $t$ é:

$$
P_{\alpha\beta}(t) = |A_{\alpha\beta}|^2 = \frac{1}{4} \sin^2 2 \theta (1 + e^{-i \frac{\Delta m^2_{21}t}{2E}}) (1 + e^{+i \frac{\Delta m^2_{21}t}{2E}})  \\
= \frac{1}{4} \sin^2 2 \theta \left(2 - 2 \cos \frac{\Delta m^2_{21}t}{2E} \right)  \\
= \sin^2 2 \theta \sin^2 \frac{\Delta m^2_{21}t}{4E}
$$

Podemos dar a probabilidade de oscilación a unha distancia $L$, que corresponde a un tempo $t$.

Nótese que os experimentos se sitúan a unha distancia $L$ da fonte de neutrinos.

$$
P_{\alpha\beta}(L) = \sin^2 2 \theta \sin^2 \frac{\Delta m^2_{21}L}{4E}
$$

Se damos unidades:

$$
P_{\alpha\beta}(L) = \sin^2 2 \theta \sin^2 \frac{1.27 \Delta m^2_{21} [\mathrm{eV}^2]L [\mathrm{km}]}{4E[\mathrm{GeV}]}
$$

Trátase dunha probabilidade de forma oscilatoria, cuxa amplitude está controlada por $\theta$ (se $\theta = 0$ non hai oscilación), e a súa frecuencia por $\Delta m^2_{21}$ (se os dous neutrinos teñen igual masa, tampouco hai oscilación).


Na fórmula anterior hai dous parámetros físicos, $\Delta m^2_{21}$ e $\theta$, e dous experimentais, $E$ e $L$. Para observar a oscilación, dados un $\Delta m^2_{12}$ e unha enerxía $E$, debemos colocar o detector a unha distancia $L$ axeitada.

Consideremos o caso de $\Delta m^2 = 2 \times 10^{-3} \; \mathrm{eV}^2$ e $\theta = 47^o$

Imos calcular a probabilidade de oscilación $P_{\alpha\beta}(L)$

*Cuestión*: A que distancia deberiamos colocar o detector para observar a oscilación máxima?

In [ ]:
theta    = np.pi * 47/180.  # ángulo de mestura (rad)
delta_m2 = 0.0025           # eV^2
ene      = 1.               # GeV
posc     = lambda L: np.sin(2*theta)**2 * np.sin(1.27 * delta_m2 * L / ene)**2

fn.plot_xy(np.linspace(0, 2000, 100), posc,
           r'$L$ (km)', r'$P_{\alpha\beta}(L)$', ylim=(0, 1.));

### Oscilacións en tres familias

Como existen tres neutrinos, hai dúas diferenzas de masas ao cadrado que controlan as fases de oscilación: $\Delta m^2_{sol}$ (polos neutrinos solares) e $\Delta m^2_{atm}$ (polos atmosféricos), cuxos valores son [PDG]:

$$
\Delta m^2_{sol} \simeq 7.5 \ 10^{-5} \; \mathrm{eV}^2, \;\;\; |\Delta m^2_{atm}| \simeq 2.5 \; 10^{-3} \mathrm{eV}^2
$$

Nótese que os neutrinos son case dexenerados en masa.

A relación entre os estados de sabor $\nu_\alpha$, con $\alpha = e, \mu, \tau$, e os estados de masa $\nu_i$ con masas $m_i$, onde $i = 1, 2, 3$, vén dada por unha matriz unitaria $U_{PMNS}$, coñecida como PMNS (Pontecorvo-Maki-Nakagawa-Sakata)

A matriz $U$ ten 3 ángulos de mestura e unha fase complexa (de xeito similar á matriz CKM).

Medíronse experimentalmente os tres ángulos [PDG], pero non a fase complexa $\delta$.

$$
\theta_{sol} \simeq 33^0, \; \theta_{atm} \simeq 47^o \; \theta_{13} \simeq 8^o
$$




### O futuro das oscilacións de neutrinos

Na próxima década, o detector Hyper-Kamiokande, [HK](https://www-sk.icrr.u-tokyo.ac.jp/en/hk/), en Xapón, e [DUNE](https://www.dunescience.org/), nos EUA, determinarán seguramente a fase $\delta$ e fixarán a nosa comprensión das oscilacións de neutrinos.

HK estudará as oscilacións $\nu_\mu \to \nu_e$ e $\bar{\nu}_\mu \to \bar{\nu}_e$ con enerxías de 0.6 GeV e unha distancia de 295 km, desde a súa produción en J-PARC ata a súa detección en HK, na mina de Kamioka.

| |
| :-- |
<img src="./imgs/leptons_HK.png" width = 400 align = "center">
|Debuxo do detector HK comparado con Notre-Dame de París|


[Para esta sección véxase e afóndese en AB10.1, AB10.2, AB10.14 e MT13]

### Bibliografía

 * [AB] Alessandro Bettini, "Introduction to Elementary Particle Physcs", Cambridge U. press. Temas 7 e 10.

 * [MT] Mark Tomsom, "Modern Particle Physics", Cambridge U. press. Temas 11, 12 e 13.

 * [PDG](https://pdg.lbl.gov/) Particle Data Group.

 * [WU] C.S. Wu et al, *Phys. Rev.* **D17** 2369 (1957)

 * [MU] M. Bardon, P. Norton, J. Peoples, A. M. Sachs and J. Lee-Franzini, *Phys. Rev.
Lett.* **14**, 449 (1965).

 * [GOL] M. Goldhaber et al, *Phys Rev* **109** 1015 (1958)

 * [KATRIN] M. Aker et al. (KATRIN Collaboration)
*Phys. Rev. Lett.* **123**, 221802 (2019)

 * [SK] Y. Ashie et al, SK Collaboration, *Phys. Rev.* **D71** 112005 (2005)
